File: Sem4/MPP/java/lab11RESTjava/Domain/src/main/java/ubb/scs/mpp/model/Angajat.java
package ubb.scs.mpp.model;



public class Angajat extends Entity<Long> {
    private Long id;
    private String username;
    private String password;
    private String mail;

    public Angajat(String username, String password, String mail) {
        this.username = username;
        this.password = password;
        this.mail = mail;
    }

    public String getUsername() {
        return username;
    }

    public void setUsername(String username) {
        this.username = username;
    }

    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }

    public String getMail() {
        return mail;
    }

    public void setMail(String mail) {
        this.mail = mail;
    }

    @Override
    public String toString() {
        return "Angajat [username=" + username + ", password=" + password + ", mail=" + mail + "]";
    }

    @Override
    public Long getId() {
        return id;
    }

    @Override
    public void setId(Long aLong) {
        this.id = aLong;
    }
}

File: Sem4/MPP/cs/lab8cs/Domain/Angajat.cs
namespace Domain;

public class Angajat:Entity<long>
{
    public Angajat(string username, string password, string mail)
    {
        Username = username;
        Password = password;
        Mail= mail;
    }
    
    public string Username { get; set; }
    public string Password { get; set; }
    public string Mail { get; set; }

    public override string ToString()
    {
        return Username+":"+Password+":"+Mail;
    }
}